dora/core/common/src/main/java/alluxio/network/protocol/databuffer/DataFileChannel.java

```diff
@@ -73,7 +73,10 @@ public void readBytes(ByteBuffer outputBuf) {
 
   @Override
   public int readableBytes() {
-    throw new UnsupportedOperationException("DataFileChannel#readableBytes is not implemented.");
+    int lengthInt = (int) mLength;
+    Preconditions.checkArgument(mLength == (long) lengthInt,
+        "size of file %s is %s, cannot be cast to int", mFile, mLength);
```

Comment:
mLength is long. Can it be printed as `%s`?

Reply:
Yes, `%s` calls `toString` of the argument.